web/src/narrow_banner.ts

```diff
@@ -264,28 +264,11 @@ function pick_empty_narrow_banner(): NarrowBannerData {
                 }
 
                 if (can_toggle_narrowed_stream()) {
-                    return {
-                        title: $t({
-                            defaultMessage:
-                                "You aren't subscribed to this stream and nobody has talked about that yet!",
-                        }),
-                        // TODO: Consider moving the button to be its own option in the template.
-                        html: $t_html(
-                            {
-                                defaultMessage: "<z-button>Subscribe</z-button>",
-                            },
-                            {
-                                "z-button": (content_html) =>
-                                    `<button class="button white rounded stream_sub_unsub_button sea-green" type="button" name="subscription">${content_html.join(
-                                        "",
-                                    )}</button>`,
-                            },
-                        ),
-                    };
+                    return default_banner;
```

Comment:
This change was discussed in https://chat.zulip.org/#narrow/stream/9-issues/topic/banner.20for.20unsubscribed.20stream.20with.20no.20messages/near/1783023.